sdk/communication/AzureCommunicationChat/Source/Token/PolicyTokenCredential.swift

```diff
@@ -24,14 +24,13 @@
 //
 // --------------------------------------------------------------------------
 
-#if canImport(AzureCore)
-    import AzureCore
-#endif
+import AzureCommunication
+import AzureCore
```

Comment:
Why you remove the #if canImport? 
I remember Brandon added them to resolve a packaging issue. But I don't remember the details. Maybe look at git blame and see when/why #if canImport was added.

Reply:
If I remember correctly is was mainly around the supporting of ObjC (or lack of ObjC support in Core). But since we are moving this to Chat there is no need for this check anymore.